chainstate/test-framework/src/framework_builder.rs

```diff
@@ -57,7 +58,7 @@ pub struct TestFrameworkBuilder {
 
 impl TestFrameworkBuilder {
     /// Constructs a builder instance with values appropriate for most of the tests.
-    pub fn new() -> Self {
+    pub fn new(_rng: &mut (impl Rng + CryptoRng)) -> Self {
```

Comment:
should we use it to generate flag for tx index?

Reply:
This is the first step.